cfgov/unprocessed/js/organisms/VideoPlayer.js

```diff
@@ -0,0 +1,190 @@
+/* ==========================================================================
+   Base Video Player Class
+   ========================================================================== */
+
+// Required modules.
+import { checkDom, setInitFlag } from '../modules/util/atomic-helpers';
+import youTubeAPI from '../modules/youtube-api';
+import EventObserver from '../modules/util/EventObserver';
+
+const BASE_CLASS = 'o-video-player';
+
+let _dom;
+let _imageDom;
+let _defaultImageURL;
+let _videoId;
+let _playBtnDom;
+let _closeBtnDom;
+let _iframeDom;
+let _player;
+
+let _playBtnClickedHandlerBinded;
+let _closeBtnClickedHandlerBinded;
+let _playVideoBinded;
+let _stopVideoBinded;
```

Comment:
That's quite a lot of `lets`. How do we feel about comma-separating them? Our linter doesn't seem to mind. [Our guidance on variable declaration](https://github.com/cfpb/development/blob/master/standards/javascript.md#variable-declaration) is somewhat confusing.

Reply:
Personally I'm against comma-separated variable declarations, and I thought the linter did mind, but maybe I didn't enforce it due to both being in the codebase. 

Consider this structure:

```
let a,
b,
c;
```
This makes it (1) unnecessarily harder to find within a code search where (e.g.) `b` is first defined, and (2) makes it easier to produce syntax errors when deleting:

```
let a,
b,
```

Or re-arranging:

```
let a,
c;
b,
```

Single line variable declarations are cleaner for these reasons, in my opinion. It's not our job to do what the minifier can do.

If lengthy variable declarations are bothersome, they may indicate the functionality they define could in part be spun off to their own module.

Reply:
Those are great points!